modules/genomic_browser/php/NDB_Menu_Filter_gwas_browser.class.inc

```diff
@@ -105,13 +105,13 @@ class NDB_Menu_Filter_GWAS_Browser extends NDB_Menu_Filter
         $this->formToFilter = array(
                                'SNP_ID'       => 'GWAS.rsID',
                                'Chromosome'   => 'genome_loc.Chromosome',
-                               'Position_BP'  => 'genome_loc.StartLoc',
-                               'Major_Allele' => 'GWAS.Major_Allele',
-                               'Minor_Allele' => 'GWAS.Minor_Allele',
+                               'BP_Position'  => 'genome_loc.StartLoc',
+                               'Major_Allele' => 'GWAS.MajorAllele',
+                               'Minor_Allele' => 'GWAS.MinorAllele',
                                'MAF'          => 'GWAS.MAF',
                                'Estimate'     => 'GWAS.Estimate',
                                'StdErr'       => 'GWAS.StdErr',
-                               'P_value'      => 'GWAS.P_value',
+                               'Pvalue'       => 'GWAS.Pvalue',
```

Comment:
I don't know what would be better but ValidFilters, Columns and formToFilter needs to be consistent.